tests/installation/partitioning_lvm.pm

```diff
@@ -75,6 +87,48 @@ sub run {
             $collect_logs = 1;
         }
     }
+    elsif (get_var("STORAGE_NG")) {
+        if ($numdisks <= 1) {
+            die "Guided workflow does not skip disk selection when only one disk!"
+                if !match_has_tag('inst-partitioning-scheme');
```

Comment:
Acually, when reading Damian Conway's "Per Best Practices" I leaned, that using negative modifiers like `unless` and `until` should be avoided as much as possible. See #13 in http://www.perlmonks.org/?node_id=989115